lib/winston-mongodb.d.ts

```diff
@@ -4,17 +4,16 @@
 // Balazs Mocsai <https://github.com/mbale>
 
 import { transports } from "winston";
-import { WinstonMongoDBTransports } from 'winston-mongodb';
+import { MongoDBTransportInstance, MongoDBConnectionOptions } from 'winston-mongodb';
 
-declare module 'winston' {
-    /**
-     * Extending transport
-     *
-     * @export
-     * @interface Transports
-     * @extends {WinstonMongoDBTransports}
-     */
-    export interface Transports extends WinstonMongoDBTransports {}
+/**
+ * Extending transport
+ */
+declare module 'winston/lib/winston/transports' {
+    interface Transports {
```

Comment:
Doesn't this interface need to be exported to augment the original `Transports` typings?

Reply:
Nope,  it can be augmented in both ways by exporting it or not. 

I saw it at https://www.typescriptlang.org/docs/handbook/declaration-merging.html#global-augmentation 
so it can work w/ or w/o exporting it.

Reply:
In that example, the augmented type is being consumed in the same ES module as it’s declared, so exporting would not be needed.

However, in testing the release containing this PR, TypeScript does not recognize the augmentation in a different module importing `winston-mongodb’ unless I prepend line 13 with `export`.

Reply:
@KeithGillette thank you for pointing this out. The fix is on its way.

Reply:
@KeithGillette can you, please, check with v5.0.7?

Reply:
Yes, I confirm that declaration merging is working properly in 5.0.7, which is great because I was apparently passing an non-existent `handleExceptions` `MongoDBConnectionOptions` which I'm not getting a typing error on.

Reply:
Thanks! :)